CRM/Utils/Check/Component/Env.php

```diff
@@ -1127,4 +1127,50 @@ public function checkAngularModuleSettings() {
     return $messages;
   }
 
+  public function checkForMultipleL10NDirs() {
+    $messages = [];
+    // This is what civi thinks is the current l10n path.
+    // Even if the site is currently en_US, we still want to do this check
+    // since they could change the language later.
+    $current_l10n = self::normalizePath(\Civi::Paths()->getPath('[civicrm.l10n]/'));
+    // use array keys instead of values to automatically dedupe paths
+    $dirs = [$current_l10n => 1];
```

Comment:
Should `$current_l10n` have the same kind of guard as the others? 

```php
if (is_dir($current_l10n)) { $dirs[$current_l10n] = 1; }
```

If it doesn't exist, then I guess it's not party to a conflict... although it is different kind of error...

Reply:
I was waffling on that. I was maybe trying to do 2 checks in one but it's not that useful.